system/lib/wasmfs/syscalls.cpp

```diff
@@ -388,6 +395,7 @@ long __syscall_mkdir(long path, long mode) {
   if (curr) {
     return -EEXIST;
   } else {
+    mode &= S_IRWXUGO | S_ISVTX;
```

Comment:
Same questions as above, plus, I think the last flag is the sticky bit - why set that?

Reply:
Added comment!